### PR TITLE
Unstoppable Domains connection

### PR DESCRIPTION
**Pull request by MarcogodoyV**
This is the last integration of UD connection, the params needed to make it work must be config [here](https://dashboard.auth.unstoppabledomains.com/connect?redirectTo=%2Fclients) following this [guide](https://docs.unstoppabledomains.com/login-with-unstoppable/login-integration-guides/login-client-configuration/). Special thanks to @kamikazebr for the help you gave me to accomplish this.

ENV vars to be added/modified (by following the guide above)
`REACT_APP_UD_CLIENT_ID`
`REACT_APP_UD_REDIRECT_URI`
`REACT_APP_UD_SCOPE`

### CODE REVIEW

File: src/connectors/index.ts [hunk 1]
```diff
@@ -32,8 +33,7 @@ export const walletConnectXDAI = new WalletConnectConnector({
     100: 'https://poa-xdai.gateway.pokt.network/v1/lb/61140fc659501900341babff'
   },
   bridge: 'https://walletconnect-relay.minerva.digital',
-  qrcode: true,
-  pollingInterval: 15000
```
Comment (kamikazebr) [inline src/connectors/index.ts:36]: Some reason to we remove the pollingInterval?

File: src/hooks/useUAuthUser.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        const uauthUser = await uauth.user()
+        if (walletAddress && walletAddress.toLowerCase() === uauthUser.wallet_address?.toLowerCase()) {
+          setUser(uauthUser)
+        } else {
+          setUser(undefined)
+        }
+        console.log('uauthUser', uauthUser)
+      } catch (error) {
+        console.log('uauthUser error', error)
+        setUser(undefined)
```
Comment (kamikazebr) [inline src/hooks/useUAuthUser.ts:25]: May we can remove the console.logs here?

**Review by kamikazebr — APPROVED** [review]
Looks good to me! I left some suggestions and comments.

**Comment by kamikazebr** [comment]
Link to test https://honeyswap-interface-taupe.vercel.app/#/swap

maybe to test you need disconnect you metamask from the site before try use UD connect.

You need also have one UD domain.

File: src/hooks/useUAuthUser.ts (hunk 2 above)
Comment (MarcogodoyV) [reply src/hooks/useUAuthUser.ts]: Done, thanks for the recommendations!

File: src/connectors/index.ts (hunk 1 above)
Comment (MarcogodoyV) [reply src/connectors/index.ts:36]: Now pollingInterval is not a valid argument for `WalletConnectConnectorArguments`, I had to delete that argument when updated we3-react library to make Uauth works well. Thanks for the review and recommendations!

File: src/connectors/index.ts [hunk 3]
```diff
@@ -42,9 +42,34 @@ export const walletConnectMATIC = new WalletConnectConnector({
...
 // mainnet only
 export const authereum = new AuthereumConnector({ chainId: 1 })
+
+export const walletconnect = new WalletConnectConnector({
+  infuraId: INFURA_PROJECT_ID,
+  qrcode: true
+})
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+console.debug('REACT_APP_UD_REDIRECT_URI', process.env.REACT_APP_UD_REDIRECT_URI!)
```
Comment (rperez89) [inline src/connectors/index.ts:57]: can we remove those logs?

Comment (MarcogodoyV) [reply src/connectors/index.ts]: Done, thank you for your review!

File: src/connectors/index.ts (hunk 1 above)
Comment (kamikazebr) [reply src/connectors/index.ts:36]: Nice

File: src/hooks/useUAuthUser.ts (hunk 2 above)
Comment (kamikazebr) [reply src/hooks/useUAuthUser.ts]: Nice

**Review by rperez89 — APPROVED** [review]

**Review by kamikazebr — APPROVED** [review]

**Pull request merged**